lib/rubocop/ast/node/mixin/method_dispatch_node.rb

```diff
@@ -8,6 +8,8 @@ module MethodDispatchNode
       extend NodePattern::Macros
       include MethodIdentifierPredicates
 
+      ACCESS_MODIFIERS = %i[public protected private module_function].freeze
```

Comment:
There's also `#private_class_method`, `#public_class_method`, `#private_constant`, and `#public_constant`. 🙂